exercises/twelve-days/README.md

```diff
@@ -28,6 +28,46 @@ On the eleventh day of Christmas my true love gave to me: eleven Pipers Piping,
 On the twelfth day of Christmas my true love gave to me: twelve Drummers Drumming, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.
 ```
 
+## Hints
+
+You need to implement the `recite` function which outputs the lyrics to
+'The Twelve Days of Christmas'. You can use the provided signature
+if you are unsure about the types, but don't let it restrict your creativity.
+
+```haskell
+recite :: Int -> Int -> [String]
+```
```

Comment:
Perhaps remove this:

The `Text` alternative is provided below, and this type signature already occurs in the stub file.

Reply:
I made this change and also tried to clean up the unordered list phrases. Each entry in the list is an extension of "you can". I hope it reads a bit more cleanly now.